### PR TITLE
0466 link for loading manifest

### PR DESCRIPTION
**Pull request by giacomomarchioro**

### CODE REVIEW

**Comment by glenrobson** [comment]
Add topic to _data/topic.yml

**Comment by glenrobson** [comment]
Hi @giacomomarchioro, the editors are really pleased to see a content state recipe and are happy with the content of the recipe but suggest some text changes. Would it be possible to make these changes by Tuesday so I can't share it again with the editors on Wednesday before sending it out to the TRC?

Rename title of recipe to: Sharing a link to open a Manifest in a specific viewer. 

# Use case
You would like to provide the user a link to click that will open a specific Manifest directly in a preselected viewer application. For example, on your web page or in an email to a user or colleague, you share a link to a IIIF Manifest for a book that will automatically open in a Mirador instance.

# Implementation notes
This type of request can be implemented using the [IIIF Content State API](https://iiif.io/api/content-state) following the [HTTP GET (Query String) Parameter](https://iiif.io/api/content-state/1.0/#initialization-mechanisms-link) section.

The Manifest URL is provided as the value of the iiif-content parameter following the viewer instance URL. Assuming the viewer can be accessed at https://example.org/viewer we can pass the `iiif-content` parameter as follows:

```
https://example.org/viewer?iiif-content=manifest-url
```

Note when the content state is a plain URI, rather than a JSON object, it must not be [content-state-encoded](https://iiif.io/api/content-state/0.9/#62-content-state-encoding-and-uri-requirements).

## Example

To open the IIIF Manifest [https://iiif.io/api/cookbook/recipe/0001-mvm-image/manifest.json](https://iiif.io/api/cookbook/recipe/0001-mvm-image/manifest.json) in the Mirador viewer hosted at [https://projectmirador.org/embed/](https://projectmirador.org/embed/), we construct the link as follows:

[https://projectmirador.org/embed/?iiif-content=https://iiif.io/api/cookbook/recipe/0001-mvm-image/manifest.json](https://projectmirador.org/embed/?iiif-content=https://iiif.io/api/cookbook/recipe/0001-mvm-image/manifest.json)

Similarly with Clover we append the `iiif-content` parameter to the viewer location as so:

[https://samvera-labs.github.io/clover-iiif/docs/viewer/demo?iiif-content=https://iiif.io/api/cookbook/recipe/0001-mvm-image/manifest.json](https://samvera-labs.github.io/clover-iiif/docs/viewer/demo?iiif-content=https://iiif.io/api/cookbook/recipe/0001-mvm-image/manifest.json)

**Comment by kirschbombe** [comment]
I noticed a few things:
- The link on the cookbook index page to this recipe isn't working
- Need to remove the period at the end of the recipe title
- Maybe move "When a content state is a plain URI rather than a JSON object, it must not be content-state-encoded" to the Restrictions section?

File: _includes/links.md [hunk 1]
```diff
@@ -70,6 +70,7 @@
 
 [0377]: {{ site.cookbook_url | absolute_url }}/recipe/0377-image-in-annotation/ "Image in annotation"
 
+[0446]: {{ site.cookbook_url | absolute_url }}/recipe/0466-link-for-loading-manifest/ "Loading a manifest with a viewer using a link"
```
[the remark this answers is not in the document]
Comment (glenrobson) [reply _includes/links.md]: Should be fixed now.

**Pull request merged**